src/NzbDrone.Core/Parser/IsoLanguages.cs

```diff
@@ -64,6 +64,10 @@ public static IsoLanguage Find(string isoCode)
                 //Lookup ISO639-2T code
                 return All.FirstOrDefault(l => l.ThreeLetterCode == langCode);
             }
+            else if (langCode.Length > 3)
+            {
+                return All.FirstOrDefault(l => l.EnglishName.ToLower() == langCode.Trim());
+            }
```

Comment:
Update to support newer subtitle file format that has full language description (English instead of Eng)